package.json

```diff
@@ -264,6 +269,8 @@
     "@types/vscode": "1.27.0",
     "chai": "4.2.0",
     "cross-env": "5.2.0",
+    "husky": "^6.0.0",
+    "lint-staged": "^11.0.0",
```

Comment:
super minor: pin these dependencies to avoid changes (similar to the others).
I was also consider unpinning these (using the `^`) now that the CICD system is working nicely 🤔 .